source/persistence/MessageController.cpp

```diff
@@ -368,3 +373,53 @@ void MessageController::printSqlError()
     qDebug() << this->lastError().driverText();
     qDebug() << this->lastError().text();
 }
```

Comment:
I don't get the reason why you changed this method. It should return the resource part of a JID from a received msgId.
Now you search for a message for a given jid.

Reply:
I rolled back: mistake